bl_version.h

```diff
@@ -0,0 +1,9 @@
+#ifndef BL_VERSION_H_INCLUDED
+#define BL_VERSION_H_INCLUDED
+
+#define VERSION_MAJOR       "00"
+#define VERSION_MINOR       "01"
+#define VERSION_PATCH       "00"
+#define VERSION_OTHERS      "feature"
```

Comment:
Please leave the others string as empty string for now `""`

Reply:
done